### PR TITLE
Hsfmin1september

### PR DESCRIPTION
**Pull request by efeyazgan**
HSF Gen meeting 1 Sepmber 2022 meeting minutes

### CODE REVIEW

File: organization/_posts/2022/2022-09-01-generators.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* MD: Follow up. I do not understand what you mean by direct event to event correlation. Unfolding only works for samples not on individual events with particles. How you do that on an event-by-event basis? 
+    * AB: slide 25; you have the events at the detector level and at the particle level. If I go through the usual chain of simulation from particle level to reco level standard MC run may times, for individual events on the RHS, I will get a distribution on the LHS. So, I can have P(SIM|GEN) or P(detector|particle) distribution. Using Bayes, theorem, I can express P(particle|detector) via P(detector|particle)xprior(particle)xprior(detector). Then, assume prior(particle), I have the other. The limitation is numerical for learning these high dimensional distributions - this is where we use the generative model. Second limitation is bias from the prior(particle) and one can use iterative approaches (which is work in progress).
```
Comment (graeme-a-stewart) [inline organization/_posts/2022/2022-09-01-generators.md:45]: This formatting breaks as the | is interprted as a sort of table separator. They need to be escaped, like:

```
P(SIM\|GEN)
```

File: organization/_posts/2022/2022-09-01-generators.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Efe Yazgan: On slide 24, you mentioned the matrix element method. So, are there studies for ML that can potentially replace that. 
+    * AB: Hopefully soon. What we are working on conceptually is that you have a detector level event and you are sampling it from a gaussian and you get the corresponding probability distribution there. This is something you can do to unfold detector level events to invert to parton level. For MEM, given an event at parton level you look at the probabilities at the ME level where you have full control but you need a transfer function that gets you from parton to detector level (in the forward direction). You can train a network to learn this but since you start from events at the detector level  first you have to sample over every potential parton level event and then calculate the probability. This is numerically exteremely difficult that is why people start with a delta function which is not so correct approximation even for leptons but for jets it is much worse. People also use gaussians which is a better approximation but with NNs one has much more flexibility and more power to capture more compelx features.
```
Comment (graeme-a-stewart) [inline organization/_posts/2022/2022-09-01-generators.md:39]: exteremely -> extremely

File: organization/_posts/2022/2022-09-01-generators.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* MD: Follow up. I do not understand what you mean by direct event to event correlation. Unfolding only works for samples not on individual events with particles. How you do that on an event-by-event basis? 
+    * AB: slide 25; you have the events at the detector level and at the particle level. If I go through the usual chain of simulation from particle level to reco level standard MC run may times, for individual events on the RHS, I will get a distribution on the LHS. So, I can have P(SIM|GEN) or P(detector|particle) distribution. Using Bayes, theorem, I can express P(particle|detector) via P(detector|particle)xprior(particle)xprior(detector). Then, assume prior(particle), I have the other. The limitation is numerical for learning these high dimensional distributions - this is where we use the generative model. Second limitation is bias from the prior(particle) and one can use iterative approaches (which is work in progress).  
+    * MD: So, you also still work on the sample. You replace the histogram by a propability distribution or inverted weights  for each of the event you can classify for the origin of the particle.
```
Comment (graeme-a-stewart) [inline organization/_posts/2022/2022-09-01-generators.md:46]: propability -> probability

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Thanks @efeyazgan - a few things to fix

**Comment by efeyazgan** [comment]
Hi @graeme-a-stewart thanks! I made the changes you requested.

**Review by graeme-a-stewart — APPROVED** [review]

**Comment by graeme-a-stewart** [comment]
Thanks @efeyazgan - next time assign the PR to me and then I should usually notice it earlier!

**Pull request merged**